src/core/execute.c

```diff
@@ -1428,6 +1428,57 @@ static int apply_protect_sysctl(Unit *u, const ExecContext *c) {
         return r;
 }
 
+static int apply_protect_kernel_modules(Unit *u, const ExecContext *c) {
+        const SystemCallFilterSet *set;
+        scmp_filter_ctx *seccomp;
+        const char *sys;
+        bool syscalls_found = false;
+        int r;
+
+        assert(c);
+
+        if (skip_seccomp_unavailable(u, "ProtectKernelModules="))
+                return 0;
+
+        seccomp = seccomp_init(SCMP_ACT_ALLOW);
+        if (!seccomp)
+                return -ENOMEM;
+
+        r = seccomp_add_secondary_archs(seccomp);
+        if (r < 0)
+                goto finish;
+
+        for (set = syscall_filter_sets; set->set_name; set++)
+                if (streq(set->set_name, "@module")) {
+                        syscalls_found = true;
+                        break;
+                }
```

Comment:
Ok done, but I think we may redo it later...

1) we may want to show filtered syscall through D-Bus per group , 2) currently we don't show system calls that are filtered through other options in D-Bus systemcallfilter property, we only show the ones that were explicitly provided, not the syscalls that we have added... I already have a fix for this one, this way it will be easy to debug since at linuxcon everyone was asking how to debug... but lets delay it for now. I only want to be consistent. ok.